keras/engine/base_layer_test.py

```diff
@@ -462,6 +462,11 @@ def __init__(self):
     self.assertEqual(self.evaluate(layer.weights[0]), 1.)
     self.assertEqual(self.evaluate(layer.weights[1]), 2.)
 
+  def test_exception_if_trainable_not_boolean(self):
+    with self.assertRaises(TypeError) as e:
```

Comment:
Any `assertRaises` should use a regex to check that the error being raised is the one you expect (and not another error upstream)